certbot/docs/install.rst

```diff
@@ -6,80 +6,49 @@ Get Certbot
    :local:
 
 
-About Certbot
-=============
 
-*Certbot is meant to be run directly on a web server*, normally by a system administrator. In most cases, running Certbot on your personal computer is not a useful option. The instructions below relate to installing and running Certbot on a server.
```

Comment:
I'm not sure how obvious this is to our audience who "just want a certificate". What do you think about leaving it in?